### PR TITLE
Edit Profile - 10/10/2019

### PR DESCRIPTION
**Pull request by SMCIND**

### CODE REVIEW

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -49,12 +53,23 @@
...
+                        <a href="#" class="dropdown-item" (click)='logout();'>Logout</a>
+                    </div>
+=======
             <ul class="navbar-nav">
                 <li class="nav-item pr-5 mt-1 ">
                         <h4 class="nav-link font-weight-bold text-dark bg-light"> Hi! {{currentCustomer.firstName}} </h4>
                 </li>
                 <li class="nav-item">
                     <a class="nav-link font-weight-bold" (click)='logout();'>LOGOUT</a>
+>>>>>>> 697d3a880909978d060cccf1910d0ecef1cfd6c5
```
Comment (saigoud1000) [inline src/app/header/header.component.html]: You need to resolve the conflicts here or else your build will fail.

File: src/app/find-business/find-business.component.html [hunk 2]
```diff
@@ -56,7 +56,22 @@
                     <div>
                     <!-- Title -->
                     <h4 class="card-title font-weight-bold mb-2">
-                        <a href="https:\\{{businessi.websiteUrl}}" target="_blank" class="uppercase">{{businessi.businessName}}</a></h4>
+                        <a href="https:\\{{businessi.websiteUrl}}" target="_blank" class="uppercase">{{businessi.businessName}}</a>
+                        <!-- <h6 class="starrating">
```
Comment (saigoud1000) [inline src/app/find-business/find-business.component.html]: remove the commented out code

Comment (SMCIND) [reply src/app/find-business/find-business.component.html]: I would be using that code for integrating rating concept

Comment (saigoud1000) [reply src/app/find-business/find-business.component.html]: Okay

**Pull request merged**